src/audio/mixer.c

```diff
@@ -372,6 +359,33 @@ static int mixer_copy(struct comp_dev *dev)
 		sources[i] = buffer_release(sources[i]);
 	}
 
+#if CONFIG_IPC_MAJOR_3
```

Comment:
hm, this was available for both IPC versions before?

Reply:
Thanks @lyakh ! Upon second thought, this does not make sense. I now applied the common silence policy for both IPC versions.